server/routes/index.ts

```diff
@@ -39,6 +40,7 @@ export default function Index(services: Services): Router {
   router.use(supportRoutes(services))
   router.use(changeLocationRoutes(services))
   router.use(changeTeamRoutes(services))
+  router.use(probationSearchRoutes(services))
```

Comment:
Should this sit inside one of the route files defined above? 
does it belong to a single area or is it shared across multiple areas?

Reply:
Shared across create and vary journeys - might be better to have generic `searchRoutes` instead?